dashboards/google-cloud-run/metadata.yaml

```diff
@@ -0,0 +1,31 @@
+sample_dashboards:
+  -
+    category: Compute
```

Comment:
nit: Cloud Run for the categories in this file.

Reply:
Good catch, fixed (and deleted the unnecessary entries in this file)